sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/TooManyParameters.cs

```diff
@@ -31,4 +31,21 @@ public interface If
         void F1(int p1, int p2, int p3);
         void F2(int p1, int p2, int p3, int p4); // Noncompliant
     }
+
+    public class MyWrongClass
+    {
+        public MyWrongClass(string a, string b, string c, string d, string e, string f, string g, string h) // Noncompliant
+        {
+        }
+    }
+
+    public class SubClass : MyWrongClass
+    {
+        // See https://github.com/SonarSource/sonar-csharp/issues/1015
+        // We should not raise when parent base class forces usage of too many args
+        public SubClass(string a, string b, string c, string d, string e, string f, string g, string h) // Compliant (base class requires them)

```

Comment:
Could add a test case for where the base is compliant but the child isn't

Reply:
Will do